src/transformState.js

```diff
@@ -6,6 +6,25 @@
  */
 function transformState(state, actions) {
   // write code here
+  actions.forEach(action => {
+    if (action['type'] === 'addProperties') {
+      Object.assign(state, action['extraData']);
+    }
+
+    if (action['type'] === 'removeProperties') {
+      action['keysToRemove'].forEach(item => {
+        if (state[item]) {
+          delete state[item];
+        }
+      });
+    }
+
+    if (action['type'] === 'clear') {
+      for (const key in state) {
+        delete state[key];
+      };
+    }
```

Comment:
Good work but to improve readibility let's rewrite it using switch/case statement.

You can also destructure action object